### ISSUE TITLE
beaconManager.isBound() and beaconManager.stopRangingBeaconsInRegion(region) are not working in Android Oreo

### ISSUE THREAD

**Issue by veerendra241** [AltBeacon/android-beacon-library#676]
### Expected behavior
        isBound() should return true if already bounded and stopRangingBeaconsInRegion()  should stop scanning and didRangeBeacons() should stop calling.

### Actual behavior
      isBound() returns true,  stopRangingBeaconsInRegion()  is not stopping scan, scan is continious even after calling this method.
### Steps to reproduce this behavior
       Library is working perfect for below Android 8 versions.
beaconManager.bind(this);
if (beaconManager.isBound(this)) // this returns false always
{
beaconManager.unbind(this);
}

///
        try {
            beaconManager.stopRangingBeaconsInRegion(region); ///this does not stops scan. didRangeBeacons calls continuesly
        } catch (RemoteException e) {
            e.printStackTrace();
        }
        
### Mobile device model and OS version
Android Oreo
 oneplus

### Android Beacon Library version
2.13.1

IMPORTANT:  This forum is reserved for feature requests or reproducible bugs with the library itself.  If you need help with using the library with your project, please open a new question on StackOverflow.com.

**Comment by veerendra241** [comment]
Can anyone please help me on this issue?

**Comment by davidgyoung** [comment]
The issue with bind appears to be a duplicate of #613.  This was actually fixed in #655, but a binary release has not been issued since that fix.  I expect a binary release to include this in the next few weeks.  


The second issue described about `stopRangingBeaconsInRegion(...)` not stopping ranging may be related to the bind issue in the code shown.   I tried to reproduce the issue using the library's  [reference application](https://github.com/AltBeacon/android-beacon-library-reference) by modifying the RangingActivity.java to stop ranging after 5 seconds by calling `stopRangingBeaconsInRegion(...)`, and the ranging callbacks stopped as expected.  Please try this same test case and verify you see the same.  If you can reproduce the problem with the reference app, please reopen this issue.

Here's the change that I made to successfully verify that `stopRangingBeaconsInRegion(...)` works on Oreo.

FROM:
```
        try {
            beaconManager.startRangingBeaconsInRegion(new Region("myRangingUniqueId", null, null, null));
        } catch (RemoteException e) {   }
```

TO:
```
        try {
            beaconManager.startRangingBeaconsInRegion(new Region("myRangingUniqueId", null, null, null));
            new Handler().postDelayed(new Runnable() {
                @Override
                public void run() {
                    try {
                        beaconManager.stopRangingBeaconsInRegion(new Region("myRangingUniqueId", null, null, null));
                    }
                    catch (RemoteException e) {   }
                }
            }, 5000);
        } catch (RemoteException e) {   }
```
